src/analysis/sensitivity.jl

```diff
@@ -82,6 +82,72 @@ function _get_cat_quantile(
     return round.(quantile(lb:ub, steps))
 end
 
+"""
+    _create_seq_store(model_spec::DataFrame, unordered_cat::Vector{Symbol}, S::Int64)
+
+Get stored bin sequences for each factor type.
+
+# Arguments
+- `model_spec` : Model specification, as extracted by `ADRIA.model_spec(domain)` or from a `ResultSet`
+- `unordered_cat` : Factors considered for sensitivity analysis of unordered categorical type.
+- `S` : Number of bins.
+"""
+function _create_seq_store(model_spec::DataFrame, unordered_cat::Vector{Symbol}, S::Int64)
+    seq_store::Dict{Symbol,Vector{Float64}} = Dict() # storage for bin sequences
+
+    # Get unique bin sequences for unordered categorical variables and store
+    for factor in unordered_cat
+        S_temp = _category_bins(model_spec[model_spec.fieldname .== factor, :])
+        seq_store[factor] = collect(0.0:(1 / S_temp):1.0)
+    end
+    # Other variables have default sequence using input S
+    seq_store[:default] = collect(0.0:(1 / S):1.0)
+
+    return seq_store
+end
+
+function _create_yax_tuple_store(
+    seq_store::Dict{Symbol,Vector{Float64}}, foi_spec::DataFrame,
+    unordered_cat::Vector{Symbol}; second_dim::Union{Dim,Vector{Any}}=[],
+)
```

Comment:
Add docstrings to this please, it's more than a few lines and it seems important.